parts/k8s/kubernetescustomscript.sh

```diff
@@ -112,6 +111,10 @@ if [[ ! -z "${MASTER_NODE}" ]]; then
     fi
 fi
 
+if [[ "${GPU_NODE}" = true ]]; then
```

Comment:
this can be `if  "${GPU_NODE}"`

Reply:
I don't trust bash:

https://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script/21210966#21210966